### ISSUE TITLE
Desync when units push eachother

### ISSUE THREAD

**Issue by SoulMuncher** [jordan-octavia-blue/Spellmasons#248]
... out of the way while moving

**Comment by SoulMuncher** [comment]
related #104

**Comment by jdoleary** [comment]
This will be complex.  It's needed to make sure units don't all just stack in a huge line but hard to syncronize

**Comment by SoulMuncher** [comment]
Was this fixed by #104 ?

**Comment by jdoleary** [comment]
No, unit collisions while walking is not managed by the forceMove system

**Comment by jordan-octavia-blue** [comment]
I'm migrating all outstanding issues into #1403 because I'm working solo and it's easier for me to manage. I'm also cleaning up a lot of old issues that are no longer relevant.  This one, I'll be taking another look at.
